app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavViewModel.kt

```diff
@@ -60,6 +61,9 @@ class RootNavViewModel @Inject constructor(
         action: RootNavAction.Internal.UserStateUpdateReceive,
     ) {
         val userState = action.userState
+
+        val currentUserOnboardingStatus =
+            userState?.activeAccount?.onboardStatus ?: OnboardingStatus.COMPLETE
```

Comment:
Ohh, you assume COMPLETE when null?

Reply:
Yes, thought process here is if it is null they created an account/logged in prior to this being in the app. So grandfathered in.

Reply:
Makes sense!